### DISCUSSION TITLE
Could pypy3 be updated to the Python 3.8 version?

### DISCUSSION THREAD

**Discussion by numbermaniac** [Homebrew/discussions#3342] [Everyday usage] [answered]
Currently if you install the [pypy3 formula](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/pypy3.rb) it installs the Python 3.7 version of PyPy. However, the page for the [current version of PyPy](https://www.pypy.org/posts/2022/03/pypy-v738-release.html) (which is the same version already in the Homebrew formula) mentions that there is also a Python 3.8 version available. It can also be seen on [their download webpage](https://www.pypy.org/download.html).

Now I get that we wouldn't want to use the Python 3.**9** version of PyPy because that's still marked as Beta, but the 3.8 version appears to be stable -- so why doesn't the formula use that instead of the Python 3.7 version?

**Comment by SMillerDev** [comment, marked as answer]
> so why doesn't the formula use that instead of the Python 3.7 version?

Because nobody made a PR for that yet.

**Reply by numbermaniac** [reply]
How hard would that be, out of curiosity? Is it simply changing the instances of "3.7" near the top of the file to "3.8", or is it a lot more work than that?
